receiver/redisreceiver/generated_component_test.go

```diff
@@ -10,12 +10,10 @@ import (
 	"github.com/stretchr/testify/require"
 	"go.opentelemetry.io/collector/component"
```

Comment:
This should resolve the failing linter
```suggestion

	"go.opentelemetry.io/collector/component"
```

Reply:
Actually, it's generated. you shouldn't change this file